ConnectorAutocadCivil/ConnectorAutocadCivil/UI/ConnectorBindingsAutocadCivil.Receive.cs

```diff
@@ -217,20 +221,30 @@ string id
             return;
           }
 
-          // convert base (or base fallback values) and store in appobj converted prop
-          if (commitObj.Convertible)
+          if (StoredObjects.TryGetValue(commitObj.OriginalId, out Base commitBaseObj))
```

Comment:
Is this just giving is things we successfully converted?

Reply:
No, it's giving the top level Speckle objects in the commit that the user is receiving. Does not record successful conversion or bake.